dietpi/dietpi-software

```diff
@@ -1199,6 +1199,15 @@ INDEX_BROWSER_TARGET=$INDEX_BROWSER_TARGET"
 		aSOFTWARE_CATX[$software_id]=8
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/cloud/#k3s'
 
+		#------------------
+		software_id=200
```

Comment:
Btw, `142` would be free (due to CouchPotato removal with v7.7). But actually I like this using the round number 🙂.